src/test/java/rotateit/controller/LabelWebhookControllerTest.java

```diff
@@ -4,24 +4,38 @@
 import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
 import org.springframework.core.io.ClassPathResource;
 import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
 import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.ResultActions;
+import rotateit.domain.github.label.LabelPayload;
+import rotateit.service.EventPublisher;
 
-import java.io.IOException;
 import java.net.URL;
 
 import static com.google.common.base.Charsets.UTF_8;
+import static org.junit.Assert.fail;
+import static org.mockito.BDDMockito.then;
+import static org.mockito.Matchers.isA;
 import static org.springframework.http.MediaType.APPLICATION_JSON;
 import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 import static org.springframework.test.web.servlet.setup.MockMvcBuilders.standaloneSetup;
 
 @RunWith(SpringJUnit4ClassRunner.class)
 public class LabelWebhookControllerTest {
+    private static final String JSON_PAYLOAD_PATH = "/github/label/exampleLabelWebhookRequest.json";
+    private static final String JSON_PAYLOAD = getContent(JSON_PAYLOAD_PATH);
```

Comment:
word `payload` is to generic, i would replace it with either `request` or `response`